UIKit_DZ/UIKit_DZ/Model/HelperStructures/Notion.swift

```diff
@@ -0,0 +1,13 @@
+// Notion.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Нотификация о каком либо упоминании или комментарии
+struct Notion: Notificatable {
+    var user: User
```

Comment:
```suggestion
    var user: User
```
дока на поля всех моделей и енамов